### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by eohulse** [rogora/hangish#56]
The application crashes on startup. It was happening on Sailfish OS 2.1 and the problem remains with the 2.2 update.

I reinstall the app but the problem continuous. When reinstalling I can go through the login menus up to typing the pin code. Then, when the app starts launching it crashes again.

If you guide me, I can provide more info for debugging.

**Comment by iotamudelta** [comment]
@eohulse I fear this app is unmaintained at this point. :-( I have the same issues and people have reported this on openrepos.

**Comment by rogora** [comment]
Hi guys, thank you for your continued interest in hangish.
Unfortunately I have to say that you are right, hangish is pretty much unmaintained now.

On the one hand, I really don't have the time I'd like to, to work on this project. It's been a long time since the last update, and there would be many things that need attention. I really wanted to go through a complete refactoring months ago, that would have been hugely beneficial to hangish in terms of stability and features. The idea was to use the official protobuf library from Google, along with the reverse engineered documentation of the API from the ever-helpful project hangups.

On the other hand, the Jolla project has never developed and grown as I hoped. I personally don't like the path they are taking right now, (I mean, selling customized versions of the OS to commercial partners), although I understand they're financial situation might be critical, and I'm not claiming they should not try to find investors in any way.
The hardware of the original Jolla phone is outdated today, and there's no real alternative. I never tried the Sailfish X project, since the idea of buying an Android phone, and paying all the royalties related to that, and consequently buying for a limited amount of time an OS that should be open source, never sounded like a good option to me. Finally, the lack of applications for Sailfish, for which IMHO one of the reasons is in the poor quality of the SDK tools, made me put my Jolla phone on the shelf.

So, to sum it up, I'd like to update and make hangish alive again, but the lack of interest from the community and the fact that Sailfish is not an option for everyday use for me, are keeping me away from spending more time (which is a precious resource, of course) on this project.
